src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -35,6 +35,12 @@
   margin-bottom: 7rem;
 }
 
+.search-results-count:focus, .schemaform-widget-wrapper textarea:focus {
```

Comment:
I didn't find a way to give textareas a class but this seems like a change that we want applied to all of the forms textarea